sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -1008,17 +1008,17 @@ case class UnixTimestamp(
     copy(timeExp = newLeft, format = newRight)
 }
 
-case class GetTimestampNTZ(
+/**
+ * Gets timestamps or dates from strings.
```

Comment:
Gets a timestamp from a string or a date (or a timestamp?). See `inputTypes` in the parent class.

Reply:
The expression here are quite messy. But I think if there are two children, it should be of (string, string)

Reply:
I mean your comment says that the expression returns either
1. timestamps from strings, or
2. dates from strings
which is wrong because the expression returns only timestamps. But a date could be an input.

Reply:
ok, updated.